### ISSUE TITLE
ingester exits with "code 137" while importing 😞 

### ISSUE THREAD

**Issue by metawops** [k0rventen/apple-health-grafana#6]
I'm on macOS 13.3.1 on a MacBook Pro 14" with M1 Pro Max CPU and 32GB RAM.
Everything looked good after I started the ingester.

However, after app. 4'30" the process exited with "code 137" after about 5 million records.:
```
apple-health-grafana-ingester-1  | inserted 5070000 records
apple-health-grafana-ingester-1  | inserted 5080000 records
apple-health-grafana-ingester-1  | inserted 5090000 records
apple-health-grafana-ingester-1 exited with code 137
```

No "failed" information of any kind was output. I guess that exiting with "code 137" is not good? Is it an error? Which one? What can I do?

**Comment by k0rventen** [comment]
Hey, seems like you're running into memory issues. I just merged [03fad4f](https://github.com/k0rventen/apple-health-grafana/commit/03fad4f59d4dd9688ae6fc6799fe14d9b6c1524f) which should greatly reduce the RAM usage of the ingested when dealing with millions+ records.

If you could pull the latest version (`docker-compose pull`) then retry the import 😄 

 I'm guessing you are using docker desktop, if the new version still doesn't complete the import, could you try to increase the amount of RAM allocated to the docker VM ?

**Comment by metawops** [comment]
Thank you so much! It worked fine now, without increasing the amount of RAM. 
Over 5.9 million records were successfully imported. 🎉🍾🥳💃🕺
Case closed. ✅
